packages/neon/neon/lib/src/pages/account_settings.dart

```diff
@@ -29,6 +29,8 @@ class AccountSettingsPage extends StatelessWidget {
                   AppLocalizations.of(context).accountOptionsRemoveConfirm(account.client.humanReadableID),
                 )) {
                   bloc.removeAccount(account);
+                  // ignore: use_build_context_synchronously
+                  const HomeRoute().go(context);
```

Comment:
Would it be possible to have this logic in the router? When the active account changes then the page switches to the home page but also pops all other routes on top.

Reply:
This is how it worked before the whole navigator/router changes (not your go_router changes).

Reply:
I think that I tried something similar but will also double check.
I think we could make a followup if it requires to much changes.

Reply:
Ok. I'd like to have this as it will prevent problems in the future if another place changes the active account but forgets to trigger the home page.

Reply:
I'm not sure I fully understand what you want.
- If we have only one logged in account and the user loggs out we want the login page to appear
- if we have multiple logged in accounts and either of them loggs out we want the settings page to be shown (just popping the top route)

or is there any case where we want the user to be navigated to the HomePage?

I think it's fairly trivial to implement in any way so I'll add it to this PR

Reply:
If there are multiple accounts and the active account gets logged out it should show the HomePage.
If there are multiple accounts and but it's not the active account that is logged out then it should just show the SettingsPage.

I think you can also argue that it should always show the SettingsPage, but this UX feels better to me as the HomePage will always be shown if the active account changes (as it happens for manual account switching).